src/spec/create_extension_spec.py

```diff
@@ -10,8 +10,8 @@ def main():
         doc='NWB extension to store pose estimation data',
         name='ndx-pose',
         version='0.1.1',
-        author=['Ryan Ly', 'Ben Dichter', 'Alexander Mathis', 'Liezl Maree'],
-        contact=['rly@lbl.gov', 'bdichter@lbl.gov', 'alexander.mathis@epfl.ch', 'lmaree@salk.edu'],
+        author=['Ryan Ly', 'Ben Dichter', 'Alexander Mathis', 'Liezl Maree', 'Chris Brozdowski'],
+        contact=['rly@lbl.gov', 'bdichter@lbl.gov', 'alexander.mathis@epfl.ch', 'lmaree@salk.edu', 'cbroz@datajoint'],
```

Comment:
Sorry! I didn't type correctly in my first review
```suggestion
        contact=['rly@lbl.gov', 'bdichter@lbl.gov', 'alexander.mathis@epfl.ch', 'lmaree@salk.edu', 'cbroz@datajoint.com'],
```

Reply:
I should have caught that - my bad!